Makefile

```diff
@@ -54,6 +54,12 @@ test:
 .PHONY: check
 check: flake8 test check-mo-files
 
+.PHONY: check-pylint
+check-pylint:
+	pylint *.py
+	pylint tcms
```

Comment:
Add `--load-plugins=pylint_django -d missing-docstring`. This will help silence some of the issues (at least for now).